src/main.rs

```diff
@@ -3,32 +3,21 @@
 use fastly::http::{header, Method, StatusCode};
 use fastly::{mime, Error, Request, Response};
 
-/// The name of a backend server associated with this service.
-///
-/// This should be changed to match the name of your own backend. See the the `Hosts` section of
-/// the Fastly WASM service UI for more information.
-const BACKEND_NAME: &str = "backend_name";
-
-/// The name of a second backend associated with this service.
-const OTHER_BACKEND_NAME: &str = "other_backend_name";
-
 /// The entry point for your application.
 ///
 /// This function is triggered when your service receives a client request. It could be used to
 /// route based on the request properties (such as method or path), send the request to a backend,
 /// make completely new requests, and/or generate synthetic responses.
 ///
 /// If `main` returns an error, a 500 error response will be delivered to the client.
+
 #[fastly::main]
-fn main(mut req: Request) -> Result<Response, Error> {
+fn main(req: Request) -> Result<Response, Error> {
     // Filter request methods...
     match req.get_method() {
         // Allow GET and HEAD requests.
         &Method::GET | &Method::HEAD => (),
 
-        // Accept PURGE requests; it does not matter to which backend they are sent.
-        m if m == "PURGE" => return Ok(req.send(BACKEND_NAME)?),
```

Comment:
Sorry to drag this up months after the fact, but why was purging removed?  Was this discussed anywhere?  We have customers that need to purge, it's good to have an example.